packages/eyes/src/generate-snapshots/filter-logic.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * This function exports a function, which is done to give the inner function (filterLogicModule) access
```

Comment:
It's clear without comments that this function binds the first argument to a different function. The question for me here is what does this code do, `generateFilteringLogic` is too abstract.